### DISCUSSION TITLE
Why is it working well with GPT-3.5 but not GPT-4 while I am a subscriber of ChatGPT plus

### DISCUSSION THREAD

**Discussion by aprilwang909** [mayooear/ai-pdf-chatbot-langchain#63] [Q&A]
As a subscriber of ChatGPT plus, I encountered a situation where it worked well with GPT-3.5 but failed to work with GPT-4. I registered OpenAI and Pinecone with the same email address. May I kindly ask what I should do to solve the problem?
Many thanks for your help in advance.

**Comment by text2sql** [comment]
you may not have yet API for gpt4. i am in the same situation.

**Comment by CK92149** [comment]
I ran into the same issue. Paid for GPT+ but the GPT4 api is something else. With GPT+ you only get GPT4 using chat.openai. the GPT4 api is a separate waitlist.

**Reply by aprilwang909** [reply]
Thank you very much for your answer. It seems that figuring out what queue you need to be in and what queue you are in is the first task in the GPT era。

**Comment by mayooear** [comment]
> As a subscriber of ChatGPT plus, I encountered a situation where it worked well with GPT-3.5 but failed to work with GPT-4. I registered OpenAI and Pinecone with the same email address. May I kindly ask what I should do to solve the problem? Many thanks for your help in advance.

You don't have access to the GPT-4 api yet. So you'd have to wait on the waitlist until it's released for you.
